src/components/Button/index.tsx

```diff
@@ -16,11 +21,14 @@ const Button: React.FC<IProps> = ({ toId, children, ...props }) => {
     props.onClick?.(e)
   }
 
-  return (
-    <ChakraButton {...props} onClick={handleOnClick}>
-      {children}
-    </ChakraButton>
-  )
+  /**
+   *  Prevent React warning that does not recognize `isSecondary` on DOM
+   *  while still sending prop to the theme config
+   */
+  const styles = useStyleConfig("Button", { ...props, isSecondary })
```

Comment:
This approach allows us to retain the Chakra Button component with all its defaults for ARIA and baseline styles that are not in the theme config.